docs/build/examples/network-inspector/step-2-run-the-react-app.md

```diff
@@ -27,7 +27,7 @@ npm ci
 npm start
 ```
 
-This starts the Filecoin network inspector app and opens the application in your browser at [https://localhost:3000](https://localhost:3000).
+This starts the Filecoin network inspector app and opens the application in your browser at `localhost:3000`.
```

Comment:
```
This starts the Filecoin network inspector app and opens the application in your browser at `localhost:3000`.
```

maybe

```
This starts and opens the Filecoin network inspector app in your browser at `localhost:3000`
```

Reply:
Again, I'm not touching copy in this PR. Literally just fixing links.